packages/webpack-cli/src/utils/autocomplete.ts

```diff
@@ -0,0 +1,186 @@
+import * as Fs from "fs";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const omelette = require("omelette");
+
+const appName = "webpack-cli";
+const autocompleteTree = {
+  build: [
+    {
+      long: "--config",
+    },
+    {
+      long: "--stats",
+    },
+  ],
```

Comment:
You can get them, we already has logic where store all commands and flags

Reply:
That will definitely make things more easy and maintainable. Please tell me how to get them.

Reply:
https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/src/webpack-cli.ts#L1160

You can move them to the static helper, please study the code before you start working on it

Reply:
I'm not sure where can I find static helper methods. Do they exist? I seem to have not seen them in project. My apologies if I am asking a dumb doubt. I just not want things to break at all.